### ISSUE TITLE
Fix XPath selector for XML content with defined encoding

### ISSUE THREAD

**Issue by peterdemin** [kibitzr/kibitzr#47]
When fetching XML with defined encoding, XPath transformer raises an exception, since it expects encoded content:
```
Traceback (most recent call last):
File "/home/drwho/kibitzr/bin/kibitzr", line 11, in <module>
sys.exit(cli())
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/core.py", line 722, in call
return self.main(args, kwargs)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/core.py", line 697, in main
rv = self.invoke(ctx)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/core.py", line 1066, in invoke
return _process_result(sub_ctx.command.invoke(sub_ctx))
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/core.py", line 895, in invoke
return ctx.invoke(self.callback, ctx.params)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/core.py", line 535, in invoke
return callback(args, kwargs)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/click/decorators.py", line 17, in new_func
return f(get_current_context(), *args, kwargs)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/cli.py", line 45, in once
sys.exit(main(once=True, log_level=ctx.obj['log_level'], names=name))
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/main.py", line 51, in main
execute_all(checkers)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/main.py", line 133, in execute_all
checker.check()
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/checker.py", line 41, in check
ok, report = self.transform(ok, content)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/transformer/factory.py", line 48, in run_pipeline
ok, content = transform(content)
File "/home/drwho/kibitzr/lib/python2.7/site-packages/kibitzr/transformer/html.py", line 72, in xpath_selector
root = etree.fromstring(html, parser=etree.HTMLParser())
File "src/lxml/lxml.etree.pyx", line 3228, in lxml.etree.fromstring (src/lxml/lxml.etree.c:79609)
File "src/lxml/parser.pxi", line 1843, in lxml.etree._parseMemoryDocument (src/lxml/lxml.etree.c:119069)
ValueError: Unicode strings with encoding declaration are not supported. Please use bytes input or XML fragments without declaration.
```

**Comment by peterdemin** [comment]
The problem with the fix is that kibitzr separates fetchers from transforms. And fetcher's output is always `unicode`. So all transforms are required to process `unicode`. But `lxml` works only with `bytes`, which is not available after fetcher is done.
So I'm going to implement a workaround - XPath selector will encode it's input to UTF-8, which will cover the most cases (as I hope). Since, you know, who's not using UTF-8 these days?

**Comment by peterdemin** [comment]
@virtadpt, should be fixed in 4.0.10.
